### PR TITLE
Create MusKulB26282.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New previouslz unknown item. I did not creat IDs for individuals prayers, should I? I do not think we have them. https://github.com/BetaMasaheft/Documentation/issues/2368

### CODE REVIEW

**Comment by eu-genia** [comment]
This could be a nice topic for someone's PhD or postdoc, a good repertory and classification of protective prayers...

File: MuseumKulturenBasel/MusKulB26282.xml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                        <persName ref="PRS14027ChristianBarth" role="bequeather"/>
+                    </person>
+                    <person>
+                        <persName ref="PRS14027ChristianBarth" role="owner"/>
+                    </person>              
+                    
+            
+            </listPerson>
+            </particDesc>
+            <textClass>
```
Comment (eu-genia) [inline MuseumKulturenBasel/MusKulB26282.xml:171]: why second list of keywords?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: MuseumKulturenBasel/MusKulB26282.xml [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1001MusKulBasel"/>
+                        <idno>Basel Museum der Kulturen iii 26282</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title type="complete" />
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply MuseumKulturenBasel/MusKulB26282.xml:31]: done

File: MuseumKulturenBasel/MusKulB26282.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply MuseumKulturenBasel/MusKulB26282.xml]: cleared

File: MuseumKulturenBasel/MusKulB26282.xml [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            </particDesc>
+            <textClass>
+                <keywords scheme="#ethioauthlist">
+             <term key="ChristianLiterature"/>
+                    <term key="BetaEsraelLiterature"/>
+                    <term key="Magic"/>
+                    <term key="ZaMa"/>
+                </keywords>
+            </textClass>
+            <langUsage><language ident="en">English</language>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply MuseumKulturenBasel/MusKulB26282.xml]: cleared

File: MuseumKulturenBasel/MusKulB26282.xml [hunk 4]
```diff
@@ -29,12 +29,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <summary/>
                         <msItem xml:id="ms_i1">
                             <title type="complete" />
-                            <textLang mainLang="gez">Protective prayer with ʾasmāt</textLang>
+                            <textLang mainLang="gez" corresp="LIT5888AsmatPrayer">Protective prayer with ʾasmāt</textLang>
```
Comment (eu-genia) [inline MuseumKulturenBasel/MusKulB26282.xml:32]: this should be `title `and not `textLang`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: MuseumKulturenBasel/MusKulB26282.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply MuseumKulturenBasel/MusKulB26282.xml]: Oh, sorry!
Corrected

**Pull request merged**